.pre-commit-config.yaml

```diff
@@ -30,7 +30,7 @@ repos:
     rev: v1.4.4
     hooks:
       - id: autopep8
-        args: ['--in-place', '--aggressive', '--aggressive', '--recursive', '--max-line-length=100', '--ignore=E501,E402,W503,E731']
+        args: ['--in-place', '--aggressive', '--aggressive', '--recursive', '--max-line-length=100', '--ignore=E501,E402,W503,W504,E731']
```

Comment:
don't require the breaking on long lines at binary operators (or something like that)... this basically allows PyCharm's version of breaking to be allowed in pre-commit.